### PR TITLE
Finalise New Group page

### PR DESCRIPTION
**Pull request by aayush-05**
image
image
image
image

Note: Work done on top of #1456

### CODE REVIEW

**Comment by aayush-05** [comment]
@nitin10s @tachyons please have a look at this

File: app/views/groups/new.html.erb [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
-  <h1 class="">NEW GROUP</h1>
-  <div class="profile-background">
-    <div>
-      Groups can be used by mentors to set projects for and give grades to students.
-      You can find the documentation <a href="https://docs.circuitverse.org/#/groups" target="_blank">here</a>.
+<div class="container">
+  <div class="row row-text">
+    <div class="col-xs col-sm col-md col-lg">
+      <h2 class="heading-submain">NEW GROUP</h2> <br>
+      <h5>Groups can be used by mentors to set projects for and give grades to students.
```
Comment (nitin10s) [inline app/views/groups/new.html.erb:7]: This is not a heading but a description so better we use <p> tag here with apt styling (don't make it look too bold)

Comment (aayush-05) [reply app/views/groups/new.html.erb:7]: Done

File: app/views/groups/new.html.erb [hunk 2]
```diff
@@ -1,15 +1,23 @@
...
-<div class="container text-center">
-  <h1 class="">NEW GROUP</h1>
-  <div class="profile-background">
-    <div>
-      Groups can be used by mentors to set projects for and give grades to students.
-      You can find the documentation <a href="https://docs.circuitverse.org/#/groups" target="_blank">here</a>.
+<div class="container">
+  <div class="row row-text">
+    <div class="col-xs col-sm col-md col-lg">
+      <h2 class="heading-submain">NEW GROUP</h2> <br>
```
Comment (nitin10s) [inline app/views/groups/new.html.erb:6]: Remove the "br" tag

Comment (aayush-05) [reply app/views/groups/new.html.erb]: Done

**Comment by satu0king** [comment]
LGTM

File: app/assets/stylesheets/layout.scss [hunk 3]
```diff
@@ -292,3 +292,56 @@ html {
...
+  margin-top: 30px;
+}
+
+.paragraph-text-justified {
+  margin: 30px 0;
+  text-align: justify;
+}
+
+//alignment
+* {
```
[the remark this answers is not in the document]
Comment (aayush-05) [reply app/assets/stylesheets/layout.scss:313]: Will be sorting layout.scss as a whole separately

**Pull request merged**